odl/space/base_tensors.py

```diff
@@ -427,7 +439,8 @@ def examples(self):
                    self.element(np.random.uniform(size=self.shape) +
                                 np.random.uniform(size=self.shape) * 1j))
         else:
-            # TODO: return something that always works, like zeros or ones?
+            # TODO(kohr-h): return something that always works, like zeros
```

Comment:
For non-numeric dtypes there is basically nothing left. What would zeros be for e.g. strings?

Reply:
I think `zeros` actually works with *any* type, meaningful or not.
```python
>>> np.zeros(2, dtype=bool)
array([False, False], dtype=bool)
>>> np.zeros(2, dtype="U2")
array(['', ''],
      dtype='<U2')
```

Reply:
I'll be damned

    >>> np.zeros(2, dtype=object)
        array([0, 0], dtype=object)

Reply:
Yeah I think that one at least makes a bit of sense. Doesn't matter for us since we don't allow `dtype=object`. I personally find `np.ones` with string type hilarious.